packages/documentation-framework/scripts/md/typecheck.js

```diff
@@ -34,13 +34,18 @@ declare module '\\*.svg' {
 const reactStylesDir = path.join(require.resolve('@patternfly/react-styles/package.json'), '../');
 const reactStyles = glob.sync(path.join(reactStylesDir, 'css/**/*.d.ts'))
   .map(f => f.replace(reactStylesDir, '@patternfly/react-styles/').replace(/\.d.ts$/, ''));
+const defaultImports = [
+  'react',
+  '@reach/router',
+  '@patternfly/react-core/next',
```

Comment:
is this not added to versions.json?

Reply:
I did not update that file. What are the expectations there and what is it used for exactly?